validator/sawtooth_validator/gossip/gossip.py

```diff
@@ -20,10 +22,40 @@
 from sawtooth_validator.protobuf import validator_pb2
 from sawtooth_validator.protobuf.network_pb2 import PeerRegisterRequest
 
+LOGGER = logging.getLogger(__name__)
+
 
 class Gossip(object):
     def __init__(self, network):
         self._network = network
+        self._identities = []
```

Comment:
My threading intuition would be that since _identities is being mutated in a Handler implementation/subclass it should be threadsafe.